sdk/storage/azure-storage-blob-batch/src/main/java/com/azure/storage/blob/batch/BlobBatchAsyncClient.java

```diff
@@ -45,10 +46,11 @@
 
     private final AzureBlobStorageImpl client;
 
-    BlobBatchAsyncClient(String accountUrl, HttpPipeline pipeline) {
+    BlobBatchAsyncClient(String accountUrl, HttpPipeline pipeline, BlobServiceVersion version) {
         this.client = new AzureBlobStorageBuilder()
             .url(accountUrl)
             .pipeline(pipeline)
+            .version(version.getVersionString())
```

Comment:
Do we want to update this `version` too?
But it falls under one of the codegen classes.

Reply:
It is not public API so don't worry about changing that, but we should get @alzimmermsft and / or @jianghaolu to review this to ensure that our builders always have a `serviceVersion` API and not `version` or any other naming convention.

Reply:
This is generated code - Swagger specifies it as "version" thus generated like this. I'm not going to try to guess this is actually the `serviceVersion` property in the code generator.